learn/advanced/asynchronous_operations.md

```diff
@@ -118,17 +140,89 @@ Had the task failed, the response would have included an `error` object:
 }
 ```
 
+If the task was canceled while it was `enqueued` or `processing`, it would have the `canceled` status:
+
+```json
+{
+  "uid": 1,
+  "indexUid": null,
+  "status": "canceled",
+  "type": "taskCancelation",
+  "canceledBy": 5,
+  "details": {
+    "matchedTasks": 1,
+    "canceledTasks": 1,
+    "originalQuery": "uid=1"
+  },
+  "error": null,
+  "duration": "PT1S",
+  "enqueuedAt": "2021-08-10T14:29:17.000000Z",
+  "startedAt": "2021-08-10T14:29:18.000000Z",
+  "finishedAt": "2021-08-10T14:29:19.000000Z"
+}
+```
+
+If the task is deleted after it was processed (`succeeded`, `failed`, or `canceled`), Meilisearch would return a [`task_not_found`](/reference/errors/error_codes.md#task-not-found) error.
+
 ### Filtering tasks
 
-You can filter tasks based on `status`, `type`, or `indexUid`. For example, the following command returns all tasks belonging to the index `movies`. Note that the `indexUid` is case-sensitive:
+You can filter tasks based on `uid`, `status`, `type`, `indexUid`, `canceledBy`, or date.
+
+#### Filter by `uid`
```

Comment:
That's what I did initially. But since users can cancel and delete tasks using these filters, I added an example for each here and linked it to the "Query parameters" tables for delete and cancel tasks endpoints.